src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+
+  if (days >= 7) {
+    return days * price - maxDiscount;
+  } else if (days >= 3) {
+    return days * price - minDiscount;
+  }
```

Comment:
Looks better this way

```suggestion
  if (days >= 7) {
    return days * price - maxDiscount;
  } 
  
  if (days >= 3) {
    return days * price - minDiscount;
  }
```

Reply:
Thank you